ui/pages/home/index.scss

```diff
@@ -176,4 +176,35 @@
       margin-left: 44px;
     }
   }
+
+  &__new-tokens-imported-notification {
+    margin-bottom: 4px;
+    margin-inline-start: 8px;
+    padding: 8px;
+
+    @media screen and (min-width: $break-large) {
+      margin-bottom: 50px;
+      margin-inline-end: 8px;
+      left: 70%;
+    }
+
+    &-title,
+    &-message {
+      text-align: left;
```

Comment:
Could `text-align: start` work better here for RTL, or should this always be on the left?